### PR TITLE
chore: add precision value

### PR DESCRIPTION
**Pull request by heliprokupetz14**

### CODE REVIEW

**Review by netanelC — CHANGES_REQUESTED** [review]
You should import the precision from the pycsw.cfg instead of setting a new env

File: charts/serving/charts/pycsw/values.yaml [hunk 1]
```diff
@@ -39,6 +39,7 @@ env:
   maxrecords: null
   logfile: '' #/var/log/pycsw/pycsw.log for innet1
   profiles: mc_3d
+  bboxPrecisionDigits: 5
```
Comment (RonitKissis) [inline charts/serving/charts/pycsw/values.yaml:42]: why 5? if the user has provided only 3 digits after the dot? or he has given 8 and i want all of them?

Comment (netanelC) [reply charts/serving/charts/pycsw/values.yaml:42]: the precision number can be bigger than the actial digits number. The function will act as she got the length number of the digits.
About the '5' value, agree

**Review by RonitKissis — APPROVED** [review]
whai whai

**Pull request merged**